setlexsem/generate/generate_prompts.py

```diff
@@ -294,8 +294,12 @@ def main(config_file, save_data, overwrite):
     # Sampler/Sets Config
     SET_TYPES = config["SET_TYPES"]
     N = replace_none(config["N"])
-    M_A = replace_none(config["M_A"])
-    M_B = replace_none(config["M_B"])
+    if "M" in config.keys():
```

Comment:
Please add a comment about why this is here.

Reply:
Added it, also i had it to 2 places, i consolidated it to read_config file so removed one of these.